tests/lib/paloaltofw_test.py

```diff
@@ -693,6 +703,15 @@ def testAcceptAction(self):
                                  "/entry[@name='test-accept-action']/action")
     self.assertEqual(x, 'allow', output)
 
+  def testPanSourceUser(self):
+    pol = policy.ParsePolicy(GOOD_HEADER_1 + PAN_SOURCE_USER, self.naming)
+    paloalto = paloaltofw.PaloAltoFW(pol, EXP_INFO)
+    output = str(paloalto)
+    x = paloalto.config.findtext(PATH_RULES + "/entry[@name='pan-source-user-term']/source-user/member[1]")
```

Comment:
I haven't checked how precise the existing tests are, but can this test be more precise.
And also includei n the test data (PAN_SOURCE_USER ON LINE 297) some non double-quoted strings?